js/about/contributionStatement.js

```diff
@@ -160,10 +174,19 @@ class ContributionStatement extends ImmutableComponent {
   }
 
   get ContributionStatementHeader () {
+    const imgStyle = StyleSheet.create({
```

Comment:
+1. If we will do so, why won't we do that for all files in a lump to avoid inconsistency? @cezaraugusto would you mind opening an issue to track that?